presto-geospatial-toolkit/src/main/java/com/esri/core/geometry/GeometryMemorySizeUtilsPackageWorkaround.java

```diff
@@ -0,0 +1,115 @@
+            MULTI_PATH_IMPL_INSTANCE_SIZE + ATTRIBUTE_STREAM_OF_DBL_INSTANCE_SIZE + ATTRIBUTE_STREAM_OF_INT32_INSTANCE_SIZE + ATTRIBUTE_STREAM_OF_INT8_INSTANCE_SIZE;
+
+    private static final int MULTI_LINE_STRING_INSTANCE_SIZE = ClassLayout.parseClass(OGCMultiLineString.class).instanceSize() +
+            ClassLayout.parseClass(Polyline.class).instanceSize() +
+            MULTI_PATH_IMPL_INSTANCE_SIZE + ATTRIBUTE_STREAM_OF_DBL_INSTANCE_SIZE + ATTRIBUTE_STREAM_OF_INT32_INSTANCE_SIZE + ATTRIBUTE_STREAM_OF_INT8_INSTANCE_SIZE;
+
+    private static final int POLYGON_INSTANCE_SIZE = ClassLayout.parseClass(OGCPolygon.class).instanceSize() +
+            ClassLayout.parseClass(Polygon.class).instanceSize() +
+            MULTI_PATH_IMPL_INSTANCE_SIZE + ATTRIBUTE_STREAM_OF_DBL_INSTANCE_SIZE + ATTRIBUTE_STREAM_OF_INT32_INSTANCE_SIZE + ATTRIBUTE_STREAM_OF_INT8_INSTANCE_SIZE;
+
+    private static final int MULTI_POLYGON_INSTANCE_SIZE = ClassLayout.parseClass(OGCMultiPolygon.class).instanceSize() +
+            ClassLayout.parseClass(Polygon.class).instanceSize() +
+            MULTI_PATH_IMPL_INSTANCE_SIZE + ATTRIBUTE_STREAM_OF_DBL_INSTANCE_SIZE + ATTRIBUTE_STREAM_OF_INT32_INSTANCE_SIZE + ATTRIBUTE_STREAM_OF_INT8_INSTANCE_SIZE;
+
+    private static final int GEOMETRY_COLLECTION_INSTANCE_SIZE = ClassLayout.parseClass(OGCConcreteGeometryCollection.class).instanceSize();
+
+    private GeometryMemorySizeUtilsPackageWorkaround() {}
+
+    public static long getEstimatedMemorySizeInBytes(OGCGeometry geometry)
+    {
+        switch (geometry.geometryType()) {
+            case "Point":
+                return POINT_INSTANCE_SIZE;
+            case "MultiPoint":
+                int pointCount = ((MultiVertexGeometry) geometry.getEsriGeometry()).getPointCount();
+                return MULTI_POINT_INSTANCE_SIZE + ENVELOPE_INSTANCE_SIZE + sizeOfDoubleArray(2 * pointCount);
+            case "LineString":
+                return LINE_STRING_INSTANCE_SIZE + getEstimatedMemorySizeInBytes((MultiPath) geometry.getEsriGeometry());
+            case "MultiLineString":
+                return MULTI_LINE_STRING_INSTANCE_SIZE + getEstimatedMemorySizeInBytes((MultiPath) geometry.getEsriGeometry());
+            case "Polygon":
+                return POLYGON_INSTANCE_SIZE + getEstimatedMemorySizeInBytes((MultiPath) geometry.getEsriGeometry());
+            case "MultiPolygon":
+                return MULTI_POLYGON_INSTANCE_SIZE + getEstimatedMemorySizeInBytes((MultiPath) geometry.getEsriGeometry());
+            case "GeometryCollection":
+                return getEstimatedMemorySizeInBytes((OGCGeometryCollection) geometry);
+            default:
+                throw new IllegalArgumentException("Unsupported geometry type: " + geometry.geometryType());
+        }
+    }
+
+    private static long getEstimatedMemorySizeInBytes(MultiPath multiPath)
+    {
+        int pointCount = multiPath.getPointCount();
+        int pathCount = multiPath.getPointCount();
```

Comment:
`getPointCount()` -> `getPathCount()`

Reply:
@nezihyigitbasi Nice catch, Nezih. Thank you!